src/validate/validate-address.js

```diff
@@ -1,4 +1,21 @@
 "use strict";
+const sha3 = require('../utils/sha3');
```

Comment:
The linter is erroring on this because `const` is not ES5 compatible.  Can you change this to `var`?  That should get CI passing, at which point I can merge.